portal/src/texts/universell-utforming/Roller.mdx

```diff
@@ -0,0 +1,73 @@
+
+Det forventes:
+- Forståelse av god informasjonsarktektur (kanskje vi burde lenke til noe i alle disse for å la folk fylle hull feks)
+- Forståelse av god fargebruk og kontrast
+- Forståelse av bevegelse og animasjon
+- Forståelse av hvordan designet blir oversatt til funksjonelle løsninger
+- Kunnskap om hvem våre brukere er og hvordan de bruker våre løsninger
+- Eksperter på brukertesting
+
+## Utvikling
+Utviklerne er ansvarlige for å oversette design og ideer til virkelighet, det innebærer å ha forståelese av hvordan designet blir i den virkelige verden på ulike skjermer, ulike dingser osv. Hvis man jobber med frontend i en eller annen form må man ta et stort ansvar for at våre løsninger skal være tilgjengelig for flest mulig. Man kan ikke gjemme seg bak "backend" eller "fullstack" merkelapp, jobber du med flater vi treffer kunder i, så er dette ditt ansvar.
+
+Det forventes:
+- Forståelse av semantisk HTML
+- Kunnskap om bruk av skjermleser og tastaturnavigasjon
+- Kunnskap om brukerpreferanser som redusert bevegelse
+- Kunnskap om responsiv CSS
+- Kunnskap om avanserte og/eller programatiske verifikasjonsverktøy
+- Automatisering av de deler av prosessen som kan automatiseres
+- Bry seg om brukeropplevelsen i sluttproduktet
+
+## Test
+Testerne er de som i størst grad representerer brukerne, derfor er det viktig at testerne evner å sette seg inn i en rekke forskjellige situasjoner og brukermønstre. Test er sikkerhetsnettet, men det teamet som er ansvarlig for kvaliteten på universell utforming, ikke test alene.
+
+Det forventes:
+- Eksperter på bruk, og hvordan våre kunder bruker, en rekke støtteverktøy, skjermleser, zoom, tastaturnavigasjon m.m.
+- Kritisk blikk på informasjonsarktektur, fargebruk og bevegelse
+- Kunnskap om bruk av verifikasjonsverktøy som Wave eller accessibilityinsights
+- Sette fokus på universell utforming igjennom daglig testing, regresjonstesting ved release og felles testfester
+
+## Produkt
+Som leder for et team eller produkt står man ansvarlig for produktet som leveres, det innebærer at man må ha et forhold til universell utforming i sitt produkt og i sitt team.
+
+Det forventes:
+- Forståelse for tiden det kreves å levere kvaliteten som forventes
+- Kunnskap om hvem våre brukere er og hvordan de bruker våre løsninger
+- Kunnskap om hvilke tiltak som gjøres for å sikre kvalitetskravene på design, utvikling og test
+- Følge opp kvalitetskrav for universell utforming på samme måte som feks krav til sikkerhet, performance og brukertilfredshet
+
+## Support
+De som jobber med support trenger å kunne støtte både våre brukere og teamene. Teamene sitter som regel et stykke unna selve brukeropplevelsen, så det kan være elementer kundeservice plukker opp som problemområder for brukerene som utviklingsteamene ikke er klar over.
```

Comment:
Kanskje bedre sånn: "Support-avdelingen bør bidra til å støtte brukere og teamene ved å informere og veilede om UU samt å melde fra om problemområder. Teamene sitter som regel et stykke unna selve brukeropplevelsen, så det kan være elementer kundeservice plukker opp som problemområder for brukerene som utviklingsteamene ikke er klar over."